src/main/java/frc/robot/commands/elevator/ManualElevatorControl.java

```diff
@@ -0,0 +1,56 @@
+//TODO: Need to figure out a way to ovverride the constant PID adjustment happening in Elevator.java subsystem to run this command
+
+
+package frc.robot.commands.elevator;
+
+import frc.robot.controls.Operator;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Elevator;
+
+public class ManualElevatorControl extends CommandBase {
+  Elevator m_elevator; 
+  double m_currentElevatorPos;
+  Operator m_operator;
+  /**
+   * This command allows for manual control of the elevator. 
+   * TODO: get manual control of all subsystems to be exclusive to a third controller
+   * @param elevator
+   */
+  public ManualElevatorControl(Elevator elevator, Operator operator) {
+    m_elevator = elevator; 
+    m_operator = operator;
+    addRequirements(m_elevator);
+  }
+  @Override
+  public void initialize() {
+    m_elevator.setEnabled(false); 
+
+  }
+  @Override
+  public void execute() {
+    if ((m_operator.getClampedThrottleValue() > 0) || (m_operator.getClampedThrottleValue() < 0)){
+      m_elevator.setEnabled(false); 
+      m_elevator.set(m_operator.getClampedThrottleValue());
+    }
+
+    if(m_operator.getClampedThrottleValue() == 0){
+      m_elevator.setEnabled(true); 
+      m_currentElevatorPos = m_elevator.getHeight(); 
+      m_elevator.setSetpoint(m_currentElevatorPos);
+    }
+
+    
+   
+  }
+
+  @Override
+  public void end(boolean interrupted) {
```

Comment:
You might want to reenable the PID and set the setpoint to the current position.